BookClubSocialNetwork/urls.py

```diff
@@ -60,13 +62,15 @@
     path('club/', views.AvailableClubsView.as_view(), name='available_clubs'),
     path('memberships/', views.MyClubMembershipsView.as_view(), name='my_club_memberships'),
     path('create/', views.clubs.CreateClubView.as_view(), name = 'create_club'),
-
-    # '''Club URLs'''
+  
     path('club/<str:club_url_name>/', views.ClubDashboardView.as_view(), name='club_dashboard'),
     path('club/<str:club_url_name>/members/', views.MembersListView.as_view(), name='member_list'),
     path('club/<str:club_url_name>/applicants/', views.ApplicantListView.as_view(), name='applicant_list'),
 
     path('club/<str:club_url_name>/edit/', views.EditClubView.as_view(), name='edit_club'),
 
+    # '''BookList URLs'''
+    path('user/<slug:username>/lists', BooklistListView.as_view(), name='booklists_list')
```

Comment:
Probably should convert this to <str:username>

Reply:
slug ensures the symbols safe with the urls?